docs/ad/analysis/swept_langmuir.rst

```diff
@@ -19,16 +19,7 @@ Example Notebooks
 API
 ---
 
-Sub-Packages & Modules
-^^^^^^^^^^^^^^^^^^^^^^
-
-.. autosummary::
-
-   floating_potential
-   helpers
-
 .. automodapi:: plasmapy.analysis.swept_langmuir
+   :noindex:
```

Comment:
`automodapi` is a "sub-class" of the `sphinx.ext.autodoc` directive `automodule` and, thus, the `automodule` options still work for `automodapi`.  The `sphinx.ext.autodoc` functionality is what sphinx uses to index the documented pacakage.  Basically this is what populates the `object.inv`, which is used by `sphinx.ext.intersphinx` to index and cross-link the documented package.  Since `.. automodule:: plasmapy.analysis.swept_langmuir` is used a few lines above here, without `:noindex:` sphinx would be trying to index the same package at two difference locations in the documentation.